README.md

```diff
@@ -2,20 +2,22 @@
 
 ## Overview
 
-The default Bazel registry for the Bzlmod external dependency system of Bazel. It is the recommended place to find and publish your favorite Bazel projects. Visit https://registry.bazel.build to check what modules are already available!
+The Bazel Central Registry (BCR) is the default registry for Bazel's external dependency system ("Bzlmod"). This is the GitHub repository backing the contents of the BCR, hosted at https://bcr.bazel.build/. For the website that allows you to search modules and see their versions conveniently, visit https://registry.bazel.build/.
```

Comment:
Since `bcr.bazel.build` works just as well for the web UI, should we just recommend that?

Reply:
I don't want to straight up recommend `bcr.bazel.build` as the website since it's just a redirect. `registry.bazel.build` is what you see in the browser's address bar, after all.